shared/actions/kbfs.desktop.js

```diff
@@ -283,22 +262,21 @@ function uninstallKBFSSagaSuccess(result: RPCTypes.UninstallResult) {
   return Saga.put(KBFSGen.createUninstallKBFSResult({result}))
 }
 
-function* openInWindows(openPath: string): Saga.SagaGenerator<any, any> {
+function* openWithCurrentMountDir(openPath: string): Saga.SagaGenerator<any, any> {
+  const goodPath = path.normalize(openPath)
   if (!openPath.startsWith(Constants.defaultKBFSPath)) {
-    throw new Error(`openInWindows requires ${Constants.defaultKBFSPath} prefix: ${openPath}`)
+    throw new Error(`openWithCurrentMountDir requires ${Constants.defaultKBFSPath} prefix: ${openPath}`)
   }
-  let goodPath = openPath.slice(Constants.defaultKBFSPath.length)
 
-  const state: TypedState = yield Saga.select()
-  let kbfsPath = state.config.kbfsPath
+  // turns '/keybase/private/alice' to 'private/alice'
+  const subPath = goodPath
+    .split(path.sep)
+    .slice(2)
+    .join(path.sep)
 
-  if (!kbfsPath) {
-    throw new Error('No kbfsPath')
-  }
+  let kbfsPath = yield Saga.select(state => state.config.kbfsPath)
```

Comment:
should keep the const state: TypedState = yield Saga.select().
Saga.select(state => something) doesn't flow type correctly so you lose typing

Reply:
Oh interesting. I went with destruction syntax initially but thought this would be more preferred. Will change back. Thanks!